ansible_collections/arista/avd/roles/eos_designs/templates/facts/switch/switch.j2

```diff
@@ -230,6 +243,9 @@ switch:
     evpn_overlay_peers:
       name: {{ bgp_peer_groups.evpn_overlay_peers.name | arista.avd.default(bgp_peer_groups['EVPN_OVERLAY_PEERS'].name, 'EVPN-OVERLAY-PEERS') }}
       password: {{ bgp_peer_groups.evpn_overlay_peers.password | arista.avd.default(bgp_peer_groups['EVPN_OVERLAY_PEERS'].password) | to_json }}
+    evpn_overlay_core:
+      name: {{ bgp_peer_groups.evpn_overlay_core.name | arista.avd.default(bgp_peer_groups['EVPN_OVERLAY_CORE'].name, 'EVPN-OVERLAY-CORE') }}
```

Comment:
no need to support the upper-case variant of the peer-group key, since this is a new model. So `name: {{ bgp_peer_groups.evpn_overlay_core.name | arista.avd.default('EVPN-OVERLAY-CORE') }}` should suffice. Same for password.